src/state/user.ts

```diff
@@ -21,8 +21,8 @@ export enum Slippages {
 }
 
 export enum Deadlines {
-  Ten = "TEN",
-  Thirty = "THIRTY",
+  Twenty = "TWENTY",
```

Comment:
Caution that these keys are persisted as cookies. Trying to read deleted values when hydrating the reducer could cause issues

Reply:
Updated to keep the old values as well.